### PR TITLE
Prepare v5.2 release

### PR DESCRIPTION
**Pull request by schweikert**

### CODE REVIEW

**Comment by schweikert** [comment]
@auerswal @gsnw I am preparing a new release. Let me know if you have any concerns / additions before I do it.

Btw. I also send an invite to add you both as collaborators on the repository, so that if you want you can also triage bugs, merge pull requests, etc.

**Comment by gsnw** [comment]
From my point of view there is nothing to be said against the creation of version 5.2

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+- Fix running in unprivileged mode (#248, thanks @sfan5)
+- Fix build issue for NetBSD/alpha (#255, thanks @0-wiz-0)
+- Fix build issue for OpenBSD/alpha (#275, thanks @gsnw)
+- Fix build warning for long int usage (#258, thanks @gsnw)
+- Fix build error with musl libc (#263, thanks @kraj)
+- Fix to guard against division by zero (#293, thanks @auerswal)
+- Decouple -a/-u effects from -c (#298, thanks @auerswal)
+- Added contrib/Dockerfile (#224, thanks @darless)
+- Remove host from Netdata chart titles (#253, thanks @ilyam8)
+- Add additional tests (#292, #297, thanks @auwerswal)
```
Comment (auerswal) [inline CHANGELOG.md:28]: There is a typo in my GitHub username in line 28 of the diff (`s/@auwerswal/@auerswal/`).

**Comment by auerswal** [comment]
I support making a release with the current set of fixes and additions. :-)

**Pull request merged**